contributors/join-us/recruiting.md

```diff
@@ -19,57 +19,127 @@ If you're considering a role at Mattermost, here are the **Top 10 Reasons to wor
 9. **Learn, Master, Teach:** Building new skills, becoming experts, and training others is core to our culture.
 10. **Friends around the world:** Build relationships with staff and contributors in dozens of countries.
 
-## Interview Stages
+In addition to this page, we also encourage anyone interested in interviewing at Mattermost to read [About Mattermost](https://handbook.mattermost.com/company/about-mattermost). 
+
+## Working and Meeting Across the Globe
+
+### Work Anywhere
```

Comment:
Instead of 'work anywhere', I wonder if we'd consider using 'Remote-first' as the section header.

"Remote-first" is one of they components of our culture and is also highlighted in the "About Company" section here: https://handbook.mattermost.com/company/about-mattermost#company

Reply:
To add, many of the concepts below (working anywhere with good internet, being explicit, emphasizing written communication, etc) is because we are remote-first, not because we are distributed (which could mean working in local offices across the world).

Reply:
I'm +1 on "remote-first".

Reply:
Completely agree, and I like this update. I think it also lends itself to a future update to further define how we behave as a remote-first team.

Reply:
I updated to include a link back to the "About Company" page. If that's off-style for a header, I can remove.